CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [8.1.0] - 2024-06-??
 
+- Fixed: MacGuffin placement respects vanilla item placement settings to not assign two items to one location.
```

Comment:
I'm unsure if we really want to phrase the changelog like this, but I also dont want the PR to block solely on it.
I feel like adding it to each affected game with `{macguffin name} placement respects vanilla item placement settings to not assign two items to one location.` would be  more understandable for end users. Because we don't have the term `macguffin` anywhere referenced and users may not necessarily be familiar what its supposed to mean.

Reply:
Then I need to think about all the games I don't play :p.
Prime 1: Artifacts
Prime 2: Sky Temple Keys
AM2R: DNA
MSR: DNA
(Dread was already fixed before)
Does it effect Cave Story?

Reply:
affects CS with puppies and a weapon/arthurs key/5hp life capsule

Reply:
What?

Reply:
What is the changelog for CS then?
`Fixed: "Weird thigns" placement respects vanilla item placement settings to not assign two items to one location.` :p ?

Reply:
I don't think it should be game specific tbh. Just say "Items placed before standard generation now respect..."

Reply:
Ok, good to go?

Reply:
ye